RosNet/RosReader/OpCode.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace RosNet.RosReader;
+
+public enum OpCode
```

Comment:
This should be internal, and it is completely fine to just place it within `RosBagReader.cs`, maybe even a private Enum, as noone will use it except for that class